src/AtomsGaffer/AtomsCrowdReader.cpp

```diff
@@ -632,7 +632,8 @@ ConstObjectPtr AtomsCrowdReader::computeSource( const Gaffer::Context *context )
 
 void AtomsCrowdReader::hashAttributes( const ScenePath &path, const Gaffer::Context *context, const GafferScene::ScenePlug *parent, IECore::MurmurHash &h ) const
 {
-    atomsSimFilePlug()->hash( h );
+    GafferScene::ObjectSource::hashAttributes( path, context, parent, h );
+    h.append( atomsSimFilePlug()->getValue() );
```

Comment:
John H: `AtomsCrowdReader::hashAttributes()` should be calling the base class `hashAttributes()` here.